Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Stage 1: Build the application
+FROM maven:3.9.6-amazoncorretto-21 as builder
```

Comment:
Opinion, but i would stick to the same image as cbioportal 
maven:3-eclipse-temurin-21